windows/devices/uwb/UwbSession.cxx

```diff
@@ -15,7 +15,7 @@
 using namespace windows::devices::uwb;
 using namespace ::uwb::protocol::fira;
 
-UwbSession::UwbSession(uint32_t sessionId, std::weak_ptr<::uwb::UwbSessionEventCallbacks> callbacks, std::shared_ptr<UwbDeviceConnector> uwbDeviceConnector, ::uwb::protocol::fira::DeviceType deviceType) :
+UwbSession::UwbSession(uint32_t sessionId, std::weak_ptr<::uwb::UwbSessionEventCallbacks> callbacks, std::shared_ptr<IUwbSessionDdiConnector> uwbDeviceConnector, ::uwb::protocol::fira::DeviceType deviceType) :
```

Comment:
As discussed offline, we'll remove the connector from here. However, I think we should complete this PR before the discussed changes should be made, so we can leave this here for now. This comment is just to document the future plan, so you can resolve it without taking any action here.